index.bs

```diff
@@ -1821,15 +1827,22 @@ Specifications should <em>not</em> use this on streams they did not create.
 <h4 id="rs-default-controller-has-backpressure" aoid="ReadableStreamDefaultControllerHasBackpressure"
 nothrow>ReadableStreamDefaultControllerHasBackpressure ( <var>controller</var> )</h4>
 
-<div class="note">
-  This method is used in the implementation of TransformStream.
-</div>
+This abstract operation is used in the implementation of TransformStream.
 
 <emu-alg>
   1. If ! ReadableStreamDefaultControllerShouldCallPull(_controller_) is *true*, return *false*.
   1. Otherwise, return *true*.
 </emu-alg>
 
+<h4 id="readable-stream-default-controller-can-close-or-enqueue" aoid="ReadableStreamDefaultControllerCanCloseOrEnqueue"
+nothrow export>ReadableStreamDefaultControllerCanCloseOrEnqueue ( <var>controller</var> )</h4>
```

Comment:
Probably don't want to export this one.

Reply:
It may be useful for specs that don't want to trigger the asserts in ControllerClose or ControllerEnqueue but also don't want to track the closed state themselves. But we don't need it right now so I have removed the export.